hoomd/operations.py

```diff
@@ -1,3 +1,10 @@
+"""Impliment a storage and management class for HOOMD-blue operations.
+
+Defines the `Operations` class which serves as the main class for storing and
+organizing the many parts of a simulation in a way that allows parts to be added
+and removed before and after _attaching_ C++ objects.
```

Comment:
We don't export the module docstring for `operations.py`, and t is an implementation detail. I put it there more for developers. I should just make a comment at the top of the file in case anyone uses this module.